xml/docker_concept.xml

```diff
@@ -0,0 +1,498 @@
+  virtual machines. However, &deng; only provides operating system-level
+  virtualization where the Linux kernel controls isolated containers.
+ </para>
+ <sect1 xml:id="sec.docker.installation.terminology_deng">
+  <title>&deng; Terminology</title>
+  <remark>toms 2019-02-04: check with Tanja, if you need the full title or
+  if "Terminology" is enough.</remark>
+  <para>
+   Before going into detail about &deng;, let us define some of the terms used:
+  </para>
+  <remark>toms 2019-02-04: two things:
+   1. Does it makes sense to sort the list alphabetically? It seems, we
+      discussed that already, but maybe ask Tanja what's her opinion on that.
+   2. I've asked Tanja, and we agreed, the term should NOT be repeated
+      in the definition. For example, in the "Registry" definition, you should
+      remove the "A registry is..." and write "Storage for already created
+      images." Same for the other terms.
+  </remark>
+
+  <variablelist>
+   <varlistentry xml:id="Docker_engine">
+    <term>&deng;</term>
+    <listitem>
+     <para>
+      &deng; is a server-client type application that performs all
+      tasks related to virtual machines. &deng; comprises the
+      following:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <formalpara>
+        <title>Daemon:</title>
+        <para>
+         The server side of &deng; manages all &docker; objects (images,
+         containers, network connections used by containers, etc.).
+        </para>
+       </formalpara>
+      </listitem>
+      <listitem>
+       <formalpara>
+        <title>REST API:</title>
+        <para>
+         Applications can use this API to communicate directly with the daemon.
+        </para>
+       </formalpara>
+      </listitem>
+      <listitem>
+       <formalpara>
+        <title>CLI Client:</title>
+        <para>
+         Enables you to communicate with the daemon. If the daemon is running
+         on a different machine than the CLI client, the CLI client can
+         communicate by using network sockets or the REST API provided by
+         &deng;.
+        </para>
+       </formalpara>
+      </listitem>
+     </itemizedlist>
+    </listitem>
+   </varlistentry>
+   <varlistentry xml:id="Docker_Image">
+    <term>Image</term>
+    <listitem>
+     <para>
+      An <emphasis>image</emphasis> is a read-only template<remark>toms 2019-02-04: 
+      template sounds a bit strange in this context.
+      On Quora.com, I've found an interesting definition:
+      "An image is a compressed, self-contained piece of software which has
+      to be unwrapped in order to use the functionality." Maybe you can add
+      it somehow.</remark> used to create a
+      <emphasis>virtual machine</emphasis> on the host server. A &docker; image is
+      made by a series of layers built one over the other. Each layer
+      corresponds to a permanent change, for example an update of an
```

Comment:
not sure what you want to say with "permanent change"? in fact, it seems a bit contradictory to me -  a change is something that happens at a given point in time (or during a transition period), how can this be permanent at the same time?

Reply:
a container image is unique. It is only used by a container engine. It is not a virtual machine. A "docker" image is a file, unique to docker and is made up of multiple layers each being read only except for the last layer.

Reply:
@taroth21 replaced "change" with "modification"